src/ReturnBookUI.java

```diff
@@ -1,87 +1,81 @@
 import java.util.Scanner;
 
-
 public class ReturnBookUI {
 
-	public static enum UI_STATE { INITIALISED, READY, INSPECTING, COMPLETED };
+	public enum UIState { INITIALISED, READY, INSPECTING, COMPLETED };
 
-	private ReturnBookControl CoNtRoL;
-	private Scanner input;
-	private UI_STATE StATe;
+	private ReturnBookControl control;
+	private Scanner systemInInput;
+	private UIState currentState;
 
-	
 	public ReturnBookUI(ReturnBookControl control) {
-		this.CoNtRoL = control;
-		input = new Scanner(System.in);
-		StATe = UI_STATE.INITIALISED;
-		control.Set_UI(this);
+		this.control = control;
+		this.systemInInput = new Scanner(System.in);
+		this.currentState = UIState.INITIALISED;
+		this.control.setUI(this);
 	}
 
-
-	public void RuN() {		
-		output("Return Book Use Case UI\n");
+	public void run() {
+		this.output("Return Book Use Case UI\n");
 		
 		while (true) {
 			
-			switch (StATe) {
+			switch (this.currentState) {
 			
 			case INITIALISED:
 				break;
 				
 			case READY:
-				String Book_STR = input("Scan Book (<enter> completes): ");
-				if (Book_STR.length() == 0) {
-					CoNtRoL.Scanning_Complete();
+				String bookScan = askQuestion("Scan Book (<enter> completes): ");
+				if (bookScan.length() == 0) {
+					this.control.completeScanning();
 				}
 				else {
 					try {
-						int Book_Id = Integer.valueOf(Book_STR).intValue();
-						CoNtRoL.Book_scanned(Book_Id);
+						int bookId = Integer.valueOf(bookScan).intValue();

```

Comment:
Need to make decision on 'ID' vs 'Id' or 'id'.
Discuss in meeting.

Reply:
As discussed "Id" will be the standard as it more closely follows the style guidelines.